### ISSUE TITLE
[V3] Warn command error

### ISSUE THREAD

**Issue by Mystard** [Cog-Creators/Red-DiscordBot#1670]
# Command bugs

#### Command name

warn

#### What cog is this command from?

warnings

#### What were you expecting to happen?

Expected a user to be warned

#### What actually happened?

User was not warned. Received both error "That is not a registered reason!" (This is a bit odd since I have allowed custom reasons) and `Error in command 'warn'. Check your console or logs for details.`

#### How can we reproduce this issue?

If I only knew. People on #1562 didn't seem to be able to.

Debian 8 Jessie, Python 3.5.3+, Red - Version 3.0.0b14, even tried updating the warning cog to the lastest version from V3/devel but no luck there either.

```
Exception in command 'warn'
Traceback (most recent call last):
  File "/home/red3/.local/lib/python3.5/site-packages/discord/ext/commands/core.py", line 62, in wrapped
    ret = yield from coro(*args, **kwargs)
  File "/home/red3/.local/lib/python3.5/site-packages/redbot/cogs/warnings/warnings.py", line 249, in warn
    "points": reason_type["points"],
KeyError: 'points'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/red3/.local/lib/python3.5/site-packages/discord/ext/commands/bot.py", line 886, in invoke
    yield from ctx.command.invoke(ctx)
  File "/home/red3/.local/lib/python3.5/site-packages/discord/ext/commands/core.py", line 514, in invoke
    yield from injected(*ctx.args, **ctx.kwargs)
  File "/home/red3/.local/lib/python3.5/site-packages/discord/ext/commands/core.py", line 71, in wrapped
    raise CommandInvokeError(e) from e
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: KeyError: 'points'
```

Tried my luck in the discord but the suggestion was to open an issue, so here it is!

**Comment by skeith** [comment]
This can be reproduced if one did not set a registered reason. User probably thought the `reason` part work the same as mod action command, accepting any value

**Comment by palmtree5** [comment]
Fixed in #1672
